### ISSUE TITLE
Leecher dont take TP 

### ISSUE THREAD

**Issue by cocaiine** [kolton/d2bot-with-kolbot#2066]
Hey guy´s can maybe someon help me with my Problem ? I setup Leader and Leecher config but my Leecher just join games but don´t takes TP´s what i make wrong ?

**Comment by mf022** [comment]
creating and joining games - is related to the starter scripts.

the in-game behavior - is related to char configuration, so choose the right options there. 
check https://github.com/blizzhackers/documentation/blob/master/kolbot/MultiBotting.md/#team-games

**Comment by cocaiine** [comment]
> creating and joining games - is related to the starter scripts.
> 
> the in-game behavior - is related to char configuration, so choose the right options there.
> check https://github.com/blizzhackers/documentation/blob/master/kolbot/MultiBotting.md/#team-games




Thank you for the answer, so does that mean I have to open my leader with d2botLead and the leech with D2Botfollow? And put the route of the two completely the same?

**Comment by mf022** [comment]
> does that mean I have to open my leader with d2botLead and the leech with D2Botfollow?

no. All depends of what run you want. Check the linked guide and the char configuration file to see the options.

Notes:
- creator of the game doesn't have to be the leader in game.
- you can have more bots in the same game doing different quests independently
- leader (boss/area scripts = true) should be the teleporter char if you set a MFTeam, with clearing of same areas, and other chars from team have activated only MFHelper script. This requires LocalChat in mode 1.
 - all bots in a game can do together Diablo + Baal where a teleporter char should be the leader (Diablo , Baal scripts)  and the others (DiabloHelper, BaalHelper). This requires LocalChat in mode 1.

**Comment by cocaiine** [comment]
> > does that mean I have to open my leader with d2botLead and the leech with D2Botfollow?
> 
> no. All depends of what run you want. Check the linked guide and the char configuration file to see the options.
> 
> Notes:
> 
> * creator of the game doesn't have to be the leader in game.
> * you can have more bots in the same game doing different quests independently
> * leader (boss/area scripts = true) should be the teleporter char if you set a MFTeam, with clearing of same areas, and other chars from team have activated only MFHelper script. This requires LocalChat in mode 1.
> * all bots in a game can do together Diablo + Baal where a teleporter char should be the leader (Diablo , Baal scripts)  and the others (DiabloHelper, BaalHelper). This requires LocalChat in mode 1.


Thank you for your detailed answer :) I will try my luck to get it going, and soory for the noob questions is my first time using the Kolbot: D
